resources/public/api/conf/1.0/application.yaml

```diff
@@ -493,485 +250,259 @@ components:
       type: object
       required:
         - eori
```

Comment:
We shouldn't have eori in the request body as we already have it in the url param
Note: We should have it in the response though